xilem_web/README.md

```diff
@@ -1,17 +1,43 @@
-# `xilem_web`
+<div align="center">
 
-This is a prototype of a potential implementation of the Xilem architecture (implementing `xilem_core`) using DOM elements as Xilem elements (unfortunately the two concepts have the same name).
+# Xilem Web
 
-The easiest way to run it is to use [Trunk] (see the examples in the `web_examples/` directory).
+</div>
+
+<!-- Close the <div> opened in lib.rs for rustdoc, which hides the above title -->
+
+</div>
+
+<div align="center">
+
+**Experimental implementation of the Xilem architecture for the Web**
+
+[![Latest published version.](https://img.shields.io/crates/v/xilem_web.svg)](https://crates.io/crates/xilem_web)
+[![Documentation build status.](https://img.shields.io/docsrs/xilem_web.svg)](https://docs.rs/xilem_web)
+[![Apache 2.0 license.](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](#license)
+
```

Comment:
```suggestion
\
```
This makes the badge lines become closer together (a line break rather than a paragraph break)